Tensile/TensileCreateLibrary.py

```diff
@@ -396,69 +397,78 @@ def buildSourceCodeObjectFiles(CxxCompiler, kernelFiles, outputPath):
 
 
 ################################################################################
-def prepAsm(kernelWriterAssembly):
-    """
-    Create and prepare the assembly directory  - called ONCE per output dir:
+def prepAsm(kernelWriterAssembly: KernelWriterAssembly, buildPath: Path):
```

Comment:
Should we test this function as a part of this PR?

Reply:
Yes, I will write some tests and add them.